src/main/java/org/kohsuke/github/GHEventPayload.java

```diff
@@ -163,22 +208,8 @@ void wrapUp(GitHub root) {
      *      check_suite event</a>
      * @see <a href="https://docs.github.com/en/rest/reference/checks#check-suites">Check Suites</a>
      */
-    @SuppressFBWarnings(
-            value = { "UWF_UNWRITTEN_PUBLIC_OR_PROTECTED_FIELD", "UWF_UNWRITTEN_FIELD", "NP_UNWRITTEN_FIELD" },
```

Comment:
most of suppressed spotbugs findings were redundant, also generalized on the class